src/antidote_app.erl

```diff
@@ -65,6 +65,13 @@ start(_StartType, _StartArgs) ->
                   ok
             end,
 
+            case application:get_env(antidote, auto_start_read_servers) of
+                {ok, true} ->
+                    %% start read servers
+                    inter_dc_manager:start_bg_processes(stable);
+                _->
+                    dont_start_read_servers
```

Comment:
this atom is not assigned to anything nor returned.
It is there just to "explain" that read servers won't be started.
In this branch of the case, nothing should be done.
Should I use an IF instead?